rel-eng/tito.props

```diff
@@ -343,6 +343,7 @@ whitelist = rubygem-angular-rails-templates
   rubygem-ffi
   rubygem-foreman-tasks
   rubygem-foreman-tasks-core
+  rubygem-fog-kubevirt
```

Comment:
I think this should be listed before `rubygem-foreman-tasks`

Reply:
done.